deluge/ui/gtk3/addtorrentdialog.py

```diff
@@ -609,7 +607,7 @@ def on_button_url_clicked(self, widget):
         response = dialog.run()
 
         if response == Gtk.ResponseType.OK:
-            url = entry.get_text().decode('utf-8')
```

Comment:
Are you testing under Python 2 or 3? I am assuming the latter from this: https://python-gtk-3-tutorial.readthedocs.io/en/latest/unicode.html#id1